macroid-core/src/main/scala/macroid/util/Effector.scala

```diff
@@ -9,8 +9,9 @@ trait Effector[F[_]] {
 }
 
 object Effector {
-  implicit object `List is Effector` extends Effector[List] {
-    def foreach[A](fa: List[A])(f: A ⇒ Any) = fa.foreach(f)
+
+  implicit def `TraversableOnce is Effector`[T[B] <: TraversableOnce[B]]: Effector[T] = new Effector[T] {

```

Comment:
What do you think about making `Effector` contravariant instead? I’ve tried this and it seemed to work for List, Vector, Seq, etc:
```scala
trait Effector[-F[_]] {
  def foreach[A](fa: F[A])(f: A ⇒ Any): Unit
}

implicit object `TraversableOnce is Effector` extends Effector[TraversableOnce] {
  override def foreach[A](fa: TraversableOnce[A])(f: A ⇒ Any): Unit = fa.foreach(f)
}
```